baseapp_chats/graphql/object_types.py

```diff
@@ -147,7 +146,10 @@ class Meta(BaseMessageObjectType.Meta):
 
 class BaseChatRoomObjectType:
     all_messages = DjangoFilterConnectionField(get_object_type_for_model(Message))
-    participants = DjangoConnectionField(get_object_type_for_model(ChatRoomParticipant))
+    participants = DjangoFilterConnectionField(
+        get_object_type_for_model(ChatRoomParticipant),
+        filterset_class=ChatRoomParticipantFilter,
```

Comment:
instead of passing a `filterset_class` here, set it as a default `filterset_class` on `BaseChatRoomParticipantObjectType.meta`, line 34, but also make sure to list `profile__target_content_type` as `fields` on your `ChatRoomParticipantFilter`

doing this way projects will be able to override this filter if necessary